test/integration/features/story_daemon.feature

```diff
@@ -0,0 +1,101 @@
+@story_daemon @darwin @linux @windows
```

Comment:
We'd be ok with testing it only on windows and darwin, for linux setup won't start the daemon so this'll fail in linux